.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,51 @@
+name: 💡 Feature Request
+description: Suggest an idea for this project
+title: "[Feature Request]: "
+labels: ["enhancement"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to suggest a new feature!
+  
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your feature request related to a problem?
+      description: A clear and concise description of what the problem is.
+      placeholder: I'm always frustrated when [...]
+    validations:
+      required: true
+  
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you'd like
+      description: A clear and concise description of what you want to happen.
+    validations:
+      required: true
+  
+  - type: textarea
+    id: alternatives
+    attributes:
+      label: Describe alternatives you've considered
```

Comment:
Maybe this can be phrased more as a question, to open the floor to folks who maybe haven't considered an alternative? Perhaps it's a completely new feature they want: i.e. " I want to clip the data using a shapefile". Although, I do see you have marked this as not required.

Reply:
Yeah definitely don't want to make this a required field, but I'm definitely open to re-wording the label there to make it sound a little more friendly.

Reply:
Probably fine as-is. I'm likely overthinking this.

Reply:
Having review it again, I actually opted in favor of removing that section entirely. I find it unlikely that most people will actually try alternatives, and if they do they'll probably put it in the additional context box.